services/debugger/destination/cache.go

```diff
@@ -0,0 +1,108 @@
+package destinationdebugger
+
+import (
+	"fmt"
+	"github.com/rudderlabs/rudder-server/config"
+	"sync"
+	"time"
+)
+
+var (
+	CacheTTL     time.Duration
+	CacheMaxSize int
+)
+
+func loadCacheConfig() {
+	config.RegisterIntConfigVariable(2, &CacheMaxSize, true, 1, "DestinationDebugger.maxEventsCacheSize")
+	config.RegisterDurationConfigVariable(720 * time.Hour, &CacheTTL, true, 1, "DestinationDebugger.maxEventsCacheTTL")
+}
+
+type cacheItem struct {
+	objs       []*DeliveryStatusT
+	lastAccess time.Time
+}
+
+type Cache struct {
+	lock sync.RWMutex
```

Comment:
change this to cacheLock

Reply:
I should reply on this one, because I had ask for this change.

It is idiomatic in to avoid name repetition, in this case the struct is already name Cache so naming the lock `cacheLock` is considered redundant. Given this a private field I think `cacheLock` doesn't help more than `lock`.

I understand that naming of things is hard and controversial, so I am not very strong on this one.

Reply:
It was previously used as c.lock inside the code so I suggested this change